packages/core/src/lib/system-setting/system-setting.service.ts

```
@@ -0,0 +1,425 @@
+		}
+
+		const metadata = getSettingMetadata(settingKey);
+		if (!metadata) {
+			// If no metadata, return as string
+			return value;
+		}
+
+		switch (metadata.type) {
+			case 'boolean':
+				const normalized = String(value).toLowerCase().trim();
+				return normalized === 'true' || normalized === '1';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)